tests/test_e2e_10_mef_eline.py

```diff
@@ -613,3 +693,649 @@ def test_080_create_and_remove_ten_circuits_ten_times(self):
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:2"}},
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:02:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:03:2"}}
+            ]
+        }
+
+        # It sets a new circuit's backup_links
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        # print(response)
+        assert response.status_code == 200
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['primary_links'][0]['endpoint_a'] == payload['primary_links'][0]['endpoint_a']
+        assert data['primary_links'][0]['endpoint_b'] == payload['primary_links'][0]['endpoint_b']
+
+    def test_155_patch_on_dynamic_backup_path_and_primary_path(self):
+        payload = {
+            "name": "my evc1",
+            "enabled": True,
+            "dynamic_backup_path": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+                "tag": {"tag_type": 1, "value": 100}
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:02:1",
+                "tag": {"tag_type": 1, "value": 100}
+            },
+            "primary_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:2"}}
+            ]
+        }
+
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        data = response.json()
+        evc1 = data['circuit_id']
+
+        time.sleep(10)
+
+        # Command to up/down links to test if back-up path is taken
+        self.net.net.configLinkStatus('s1', 's2', 'down')
+
+        # Wait just a few seconds to give time to the controller receive and process the linkDown event
+        time.sleep(10)
+
+        current_path = [{"endpoint_a": {"id": "00:00:00:00:00:00:00:01:4"},
+                         "endpoint_b": {"id": "00:00:00:00:00:00:00:03:3"}},
+                        {"endpoint_a": {"id": "00:00:00:00:00:00:00:03:2"},
+                         "endpoint_b": {"id": "00:00:00:00:00:00:00:02:3"}}]
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+
+        paths = []
+        for _path in data['current_path']:
+            paths.append({"endpoint_a": {"id": _path['endpoint_a']['id']},
+                          "endpoint_b": {"id": _path['endpoint_b']['id']}})
+
+        # Command to up/down links to test if back-up path is taken
+        self.net.net.configLinkStatus('s1', 's2', 'up')
+
+        assert paths == current_path
+
+    def test_160_patch_on_dynamic_backup_path_and_empty_primary_and_backup_path(self):
```

Comment:
here also does not patch the EVC

Reply:
Resolved with commit [748a6ad](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/748a6ad741f70941e4528b147cc4c844214d6e95)